scripts/cxg_admin.py

```diff
@@ -185,6 +185,31 @@ def create_cxg_artifacts(ctx):
                     s3_uri=s3_uri,
                 )
 
+@cli.command()
+@click.pass_context
+def migrate_schema_version(ctx):
+    """
+    Populates `schema_version` for each existing dataset. Since the schema version only exists
+    in the cxg file and we don't want to open them, we will call the cellxgene explorer endpoint
```

Comment:
where does Explorer get the version from? Does it not have to open the files?

Reply:
Yea, we are just hitting prod and making it do the work